### PR TITLE
Why choose us section animation with Vanilla tilt

### PR DESCRIPTION
**Pull request by bhavyanarnoli**
## Description 📝

#329
## Changes 💬
 vanilla tilt only in the Landingpage

## Screenshots (if applicable)

image
image

## Additional Notes 📒

only commiting changed files

### CODE REVIEW

**Comment by bhavyanarnoli** [comment]
@PiyushKalyanpy i have done all the changes pls see my pr

**Comment by bhavyanarnoli** [comment]
@lalitmee

**Comment by bhavyanarnoli** [comment]
i have not made any change in firebase just one line got added by mistake, please conside my pr

**Comment by bhavyanarnoli** [comment]
done under gssoc

**Review by PiyushKalyanpy — APPROVED** [review]

**Comment by bhavyanarnoli** [comment]
@PiyushKalyanpy why cant i see the conflicts

**Comment by bhavyanarnoli** [comment]
ill do it in a while pls don't close the pull request

File: Components/LandingPage/WhyChooseUs.tsx [hunk 1]
```diff
@@ -1,19 +1,48 @@
...
 
 const WhyChooseUs = () => {
   return (
     <div id="whychooseus" className="px-20 mt-40 dark:bg-neutral-950">
-      <h1 className="text-4xl text-center font-semibold font-archivo text-slate-900 dark:text-zinc-50 p-4 mb-10">{`Why Choose Us`}</h1>
-      <div className="grid grid-cols-6  gap-8 w-full items-center">
-        <div className="col-span-4 grid grid-cols-3 gap-8 ">
+      <h1 className="text-4xl text-center font-semibold font-archivo text-slate-900 text-black dark:text-zinc-50 p-4 mb-10">{`Why Choose Us`}</h1>
+      <div className="grid grid-cols-6 gap-8 w-full items-center">
+        <div className="col-span-4 grid grid-cols-3 gap-8 backgroundColor:grey dark : backgroundColor: #F7EDE3">
```
Comment (lalitmee) [inline Components/LandingPage/WhyChooseUs.tsx:12]: What are we trying to do here?

**Review by lalitmee — CHANGES_REQUESTED** [review]

**Comment by PiyushKalyanpy** [comment]
Hey there, 
We are closing this pull request as the changes we requested were not implemented. Please create a new PR once you implement the changes.

**Comment by PiyushKalyanpy** [comment]
Hey @bhavyanarnoli , your valuable contribution is appreciated, and we've merged the PR after resolving conflicts. For your next contribution, please check #466 to push only the required file. Also, there are new issues opened; feel free to check them out.

**Comment by bhavyanarnoli** [comment]
thank you!

**Pull request merged**